cobalt/build/configs/android_common.gn

```diff
@@ -7,3 +7,5 @@ treat_warnings_as_errors = false
 # Overriding flag from //media/media_options.gni.
 # Cobalt doesn't use //third_party/ffmpeg.
 media_use_ffmpeg = false
+# Uncomment to update android_aar_prebuilt info file.
```

Comment:
Maybe add a reference to the original documentation: https://source.chromium.org/chromium/chromium/src/+/main:build/config/android/rules.gni;l=4245?q=build%2Fconfig%2Fandroid%2Frules.gni&ss=chromium%2Fchromium%2Fsrc

Reply:
otherwise LGTM from build perspective

Reply:
great finding, I did not see this.

Reply:
I put it in the description of the PR.